package.json

```diff
@@ -4,7 +4,7 @@
     "displayName": "Spring Boot Snippets",
     "description": "Useful snippets for Spring Boot projects.",
     "version": "1.0.1",
-    "icon": "images/spring-boot.png",
+    "icon": "images/spring-snippets.png",
```

Comment:
changing the icon.

Reply:
I do like the image, but for my VS extensions I prefer not to use official logos to avoid infringing on trademarks.

Reply:
Oh! I understand! I really don't have any ideas of what the legit logo would be. But if you have any ideas about the logo I will be happy to help.